tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_participants_under_18years.py

```diff
@@ -0,0 +1,150 @@
+"""
+Integration test for remove_participants_under_18years module
```

Comment:
This file should be renamed `remove_participants_under_18years_test.py ` to properly run the integration test